### DISCUSSION TITLE
Cannot call Codespaces even though I selected it when creating an assignment.

### DISCUSSION THREAD

**Discussion by sskeme** [community/community#112907] [GitHub Education]
### Hi everyone,

Hi, I'm having trouble calling Codespaces after creating an assignment in GitHub Classroom. Specifically, the generated assignment says "Don't use an online IDE" even though I've selected GitHub Codespaces as shown below.

![a](https://github.com/community/community/assets/154963758/ceceac82-dd2a-482f-aa27-98243423e542)

![b](https://github.com/community/community/assets/154963758/20d2f9d8-e9ec-4fdd-bd83-12dd335bfd04)

It was working fine about 10 days ago.

**Comment by TRESPADERNE** [comment]
Same problem

**Comment by serefakbogfa** [comment]
Double-Check Assignment Settings: After creating the assignment, review the assignment settings to ensure that GitHub Codespaces is selected as the designated environment for students. Look for any options related to IDE or environment settings and verify that Codespaces is chosen

**Reply by serefakbogfa** [reply]
r

**Reply by sskeme** [reply]
Thanks for your reply. I've double-checked it several times. I used the same approach for 7 assignments this year, and it worked well. The problem started occurring in mid-March 2024. 

Is there anything I need to check?

**Comment by samus-aran** [comment]
Hi @sskeme  👋

We are happy to update that we have deployed a fix for this. Can you please re-try this and confirm that it works as expected 👍
